google_permissions/other_perms.tf

```diff
@@ -0,0 +1,127 @@
+/*
+// This file is used to add additional permissions to the project. The idea is that
+// there are a set of pre-cleared permissions available for use, and that we can
+// add them to the project as needed by adding them to the `other_permissions` list.
+*/
+
+//
+// key: bq_job_user
```

Comment:
Style nit: There are a couple different styles of comments used here, especially between two options available for single line comments (`#`, `//`). For consistency I'd suggest using one.